core/lib/spree/core/controller_helpers/common.rb

```diff
@@ -11,12 +11,17 @@ module Common
           helper_method :title
           helper_method :title=
           helper_method :accurate_title
+          helper_method :plural_resource_name
 
           layout :get_layout
 
           before_action :set_user_language
         end
 
+        def plural_resource_name(resource_class)
```

Comment:
Can we move this method into the `admin/resource_controller` instead? This module gets included in many (even frontend) controllers.